test-scripts/test-vision-svc.py

```diff
@@ -0,0 +1,116 @@
+from viam.components.camera import Camera
+
+API_KEY = ''
+API_KEY_ID = ''
+MACHINE_ADDRESS = ''
+
+async def connect():
+    opts = RobotClient.Options.with_api_key(
+        # Replace "<API-KEY>" (including brackets) with your machine's api key
+        api_key=API_KEY,
+        # Replace "<API-KEY-ID>" (including brackets) with your machine's api key id
+        api_key_id=API_KEY_ID
+    )
+    return await RobotClient.at_address(MACHINE_ADDRESS, opts)
+
+async def main():
+    machine = await connect()
+
+    # detector
+    vision_1 = VisionClient.from_robot(machine, "my_detector")
+    vision_1_return_value = await vision_1.get_properties()
+    print(f"my_detector get_properties return value: {vision_1_return_value}")
+
+    camera_name = "cam"
+
+    # Grab the detector you configured on your machine
+    my_detector = VisionClient.from_robot(machine, "my_detector")
+    # Get detections from the next image from the camera
+    detections = await my_detector.get_detections_from_camera(camera_name)
+    print(detections)
+
+
+    # Grab camera from the machine
+    cam1 = Camera.from_robot(machine, "cam")
+
+    # Get the detector you configured on your machine
+    my_detector = VisionClient.from_robot(machine, "my_detector")
+
+    # Get an image from the camera
+    img = await cam1.get_image()
+
+    # Get detections from that image
+    detections = await my_detector.get_detections(img)
+
+
+
+
+    # classifier
+    vision_2 = VisionClient.from_robot(machine, "my_classifier")
+    vision_2_return_value = await vision_2.get_properties()
+    print(f"vision-1 get_properties return value: {vision_2_return_value}")
+
+    # Grab the classifier you configured on your machine
+    my_classifier = VisionClient.from_robot(machine, "my_classifier")
+
+    # Get the 2 classifications with the highest confidence scores from the next image from the camera
+    classifications = await my_classifier.get_classifications_from_camera(
+        "cam", 2)
+    print(classifications)
+
+    # Grab camera from the machine
+    cam1 = Camera.from_robot(machine, "camera-1")
+
+    # Get the classifier you configured on your machine
+    my_classifier = VisionClient.from_robot(machine, "my_classifier")
+
+    # Get an image from the camera
+    img = await cam1.get_image()
+
+    # Get the 2 classifications with the highest confidence scores
+    classifications = await my_classifier.get_classifications(img, 2)
+
+
+
+    # segmenter
+    vision_3 = VisionClient.from_robot(machine, "my_segmenter")
+    vision_3_return_value = await vision_2.get_properties()
+    print(f"vision-3 get_properties return value: {vision_3_return_value}")
+
+
+    import numpy as np
+    import open3d as o3d
+
+    # Grab the object segmenter you configured on your machine
+    my_segmenter = VisionClient.from_robot(machine, "my_segmenter")
+    # Get the objects from the camera output
+    # objects = await my_segmenter.get_object_point_clouds("cam")
```

Comment:
again, why commenting out this method? guessing on purpose as in both code samples. did it not work?

Reply:
I have no camera that returns point clouds, I've tested enough to know we have to pass in a camera name but can't test the rest. I'll uncomment for the PR though